solidity/test/LiquidityProtectionStateless.js

```diff
@@ -31,29 +34,33 @@ function assertAlmostEqual(actual, expected) {
 
 describe('LiquidityProtectionStateless', () => {
     before(async () => {
+        const owner = defaultSender;
         const governor = accounts[1];
 
         const networkToken = await DSToken.new('BNT', 'BNT', 18);
         const networkTokenGovernance = await TokenGovernance.new(networkToken.address);
-        await networkTokenGovernance.grantRole(roles.ROLE_GOVERNOR, governor);
+        await networkTokenGovernance.grantRole(ROLE_GOVERNOR, governor);
         await networkToken.transferOwnership(networkTokenGovernance.address);
         await networkTokenGovernance.acceptTokenOwnership();
 
         const govToken = await DSToken.new('vBNT', 'vBNT', 18);
         const govTokenGovernance = await TokenGovernance.new(govToken.address);
-        await govTokenGovernance.grantRole(roles.ROLE_GOVERNOR, governor);
+        await govTokenGovernance.grantRole(ROLE_GOVERNOR, governor);
         await govToken.transferOwnership(govTokenGovernance.address);
         await govTokenGovernance.acceptTokenOwnership();
 
+        const checkpointStore = await CheckpointStore.new({ from: owner });
         liquidityProtection = await LiquidityProtection.new(
```

Comment:
Sender (`from`) not important here?

Reply:
Not really. I mean, it's a matter of style. I like to always explicitly specify who is sending a transaction when it matters/permissions, but in both of the cases, since the owner is the defaultSender anyway - it won't matter